andela/mazus-ah-backend
# #167617811 Converts ID Column To UUID

### What does this PR do?
It converts the Id column to UUIDs

#### Description of Task to be completed?
Convert id field to UUID in all database migration files
Adds id property to the objects in seeder files
Created a dev migrate script in package.json
Refactored profile controllers to use the UUID
Refactored some profile controller tests
Refactored validation middleware

#### How should this be manually tested?
> 1. Fetch this branch on the terminal using 
> `git fetch origin ch-id-uuid-167617811`
> 2. Install dependencies using `npm install`
> 3. Start the server with `npm run server`
> 4. On Postman, sign up by making a POST request to /api/v1/auth/signup.
> 5. The response user object should have a UUIDv4 as the Id
> 

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167617811](https://www.pivotaltracker.com/story/show/167617811)

#### Screenshots (if appropriate)
https://drive.google.com/file/d/1WvpThSStVdJSvJcqHWQv11xyoekdVCBw/view?usp=sharing
